app/navigation/nonsipp/MoneyBorrowedNavigator.scala

```diff
@@ -65,7 +65,7 @@ object MoneyBorrowedNavigator extends JourneyNavigator {
         (
           for {
             indexes <- userAnswers
-              .map(BorrowedAmountAndRatePages(srn))
+              .map(LenderNamePages(srn))
```

Comment:
Misc bugfix, unrelated to statuses - using 1st page in journey to build indexes instead of 3rd page.